react/Button/Button.less

```diff
@@ -75,10 +88,33 @@
   .buttonColor(@sk-black, @sk-mid-gray-light);
 }
 
+.root_isGhost,
 .root_isTransparent {
-  color: @sk-link;
   background-color: transparent;
   box-shadow: none;
+}
+
+.root_isGhost {
+  border: 2px solid;
+  font-weight: @sk-bold;
+  line-height: 0;
+}
+
+.root_isBlue.root_isGhost {
+  .ghostColor(@sk-link)
+}
+
+.root_isWhite.root_isGhost {
```

Comment:
Why does this class not use the `.ghostColor` mixin?

Reply:
Because it's not a `colored` ghost button, it's white and it's different